### PR TITLE
Add limited state

### PR DESCRIPTION
**Pull request by efernandez**
This PR is on top of #21 

This includes the limited velocities into the **controller state**, which was missed in #12 

@paulbovbel @mikepurvis @afakihcpr

### CODE REVIEW

File: diff_drive_controller/src/diff_drive_controller.cpp [hunk 1]
```diff
@@ -801,14 +826,14 @@ namespace diff_drive_controller
...
 
-          state_pub_->msg_.error_estimated.positions[i] =
-            state_pub_->msg_.desired.positions[i] - state_pub_->msg_.actual_estimated.positions[i];
-          state_pub_->msg_.error_estimated.velocities[i] =
-            state_pub_->msg_.desired.velocities[i] - state_pub_->msg_.actual_estimated.velocities[i];
-          state_pub_->msg_.error_estimated.accelerations[i] =
-            state_pub_->msg_.desired.accelerations[i] - state_pub_->msg_.actual_estimated.accelerations[i];
-          state_pub_->msg_.error_estimated.effort[i] =
-            state_pub_->msg_.desired.effort[i] - state_pub_->msg_.actual_estimated.effort[i];
+          state_pub_->msg_.error_side_average.positions[i] =
```
Comment (efernandez) [inline diff_drive_controller/src/diff_drive_controller.cpp:833]: This was a bug from #12

**Comment by paulbovbel** [comment]
LGTM

**Comment by afakihcpr** [comment]
:+1:

**Comment by efernandez** [comment]
#21 was closed and merged in #30 
Closing this and creating a new PR to merge it

**Pull request closed without merging**